core/whistle_number_manager-1.0.0/include/wh_number_manager.hrl

```diff
@@ -73,7 +73,7 @@
                                        ,<<"failover">> ,<<"prepend">>
                                       ]).
 
--define(WNM_DB_PREFIX_L, "numbers/").
+-define(WNM_DB_PREFIX_L, "numbers/+").
```

Comment:
@k-anderson there are some Dbs numbers/inum, numbers/managed numbers/local that were being crawled without reason. i understand your point here but e164 is meant to be +...